web/package.json

```diff
@@ -13,6 +13,7 @@
     "url": "git+https://github.com/odrusso/eduace.git"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.10",
```

Comment:
Not sure why so many packages got updated :/
Just did added some dev packages lol